src/cplscheme/tests/DummyCouplingScheme.hpp

```diff
@@ -227,17 +228,17 @@ class DummyCouplingScheme final : public CouplingScheme {
 private:
   mutable logging::Logger _log{"cplscheme::tests::DummyCouplingScheme"};
 
-  /// @brief Number of iterations performed per timestep. 1 --> explicit.
+  /// @brief Number of iterations performed per time window. 1 --> explicit.
   int _numberIterations;
 
-  /// @brief Performed iterations in the current timestep.
+  /// @brief Performed iterations in the current time window.
   int _iterations = 0;
 
-  /// @brief Maximal number of timesteps to be performed.
-  int _maxTimesteps;
+  /// @brief Maximal number of time windows to be performed.
+  int _maxTimeSteps;
```

Comment:
```suggestion
  /// @brief Maximal number of time windows to be performed.
  int _maxTimeWindows;
```
which has an effect on more locations.

Reply:
Fixed in e8a401a0b7926e1e70be2060e2104467cbea3dd0